modules/virt-adding-secret-configmap-service-account-to-vm.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-managing-configmaps-secrets-service-accounts.adoc
+
+[id="virt-adding-secret-configmap-service-account-to-vm_{context}"]
+
+= Adding a Secret, ConfigMap, or Service Account to a virtual machine
+
+Add a Secret, ConfigMap, or Service Account to a virtual machine by using the {product-title} web console.
+
+.Prerequisites
+
+* You must have at least one available Secret, ConfigMap, or Service Account
+configured in your {product-title} cluster.
+
+* The virtual machine must be powered off.
+
+.Procedure
+
+. Navigate to the *Virtualization* -> *Virtual Machine Details* page.
```

Comment:
The navigations should be Virtualization -> Virtual Machine tabs -> select a virtual machine to open it's overview page -> Click Environment tab.